src/Description/SonataEnhancer.php

```diff
@@ -118,6 +120,29 @@ public function supports(PuliResource $resource)
         // sonata has dependency on ClassUtils so this is fine.
         $class = ClassUtils::getClass($payload);
 
-        return $this->pool->hasAdminByClass($class);
+        return null !== $this->getAdminByClass($class);
+
+    }
+
+    private function getAdminByClass($class)
+    {
+        if (isset($this->adminMap[$class])) {
```

Comment:
please use array_key_exists. i prefer that in general as it avoids weird errors when e.g. the adminMap field or $class are not defined, but in this case it does not work at all because when the array field is `null`, the php gods decided that isset returns false: https://3v4l.org/6vCgE